semgrep-core/src/targeting/List_files.mli

```diff
@@ -0,0 +1,38 @@
+(*
+   List files recursively in a safe, efficient, and portable manner.
+
+   Replaces the functions in pfff/commons that use external unix commands.
+*)
+
+type path = string
```

Comment:
maybe add a comment about why we don't use FPath.t (yet).

Reply:
I don't think explaining everything we're not using makes much sense. I introduced 3 different modules that use `type path` independently. I'm all for using `FPath` but it needs to be revised so it's independent of the state of the file system (among other things).